azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/CallingFragment.kt

```diff
@@ -155,6 +155,6 @@ internal class CallingFragment(
     }
 
     private fun requestCallEnd() {
-        confirmLeaveOverlayView.showHangupOverlay()
+        viewModel.getConfirmLeaveOverlayViewModel().show()
```

Comment:
How about we will create a method CallingViewModel.requestCallEnd() and rename  ConfirmLeaveOverlayViewModel.show() to ConfirmLeaveOverlayViewModel.requestExitConfirmation()

Then from here: CallingFragment.requestCallEnd() call viewModel.requestCallEnd() which will call confirmLeaveOverlayViewModel.requestExitConfirmation()